### PR TITLE
Update manta modules

### PR DESCRIPTION
**Pull request by maxulysse**
## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

File: modules/nf-core/manta/tumoronly/main.nf [hunk 1]
```diff
@@ -33,8 +33,9 @@ process MANTA_TUMORONLY {
     configManta.py \
         --tumorBam $input \
         --reference $fasta \
+        --runDir manta \
         $options_manta \
-        --runDir manta
+        $args
 
     python manta/runWorkflow.py -m local -j $task.cpus
```
Comment (FriederikeHanssen) [inline modules/nf-core/manta/tumoronly/main.nf:40]: then here too?

Comment (maxulysse) [reply modules/nf-core/manta/tumoronly/main.nf:40]: no I don't think we can pass args to this one

Comment (FriederikeHanssen) [reply modules/nf-core/manta/tumoronly/main.nf:40]: ah never mind

Comment (FriederikeHanssen) [reply modules/nf-core/manta/tumoronly/main.nf:40]: sorry need more coffee

**Review by FriederikeHanssen — APPROVED** [review]

**Comment by asp8200** [comment]
Could we perhaps add a pytest covering the bug fixed here?

**Pull request merged**